### ISSUE TITLE
i can't change visibility of exo_ffwd and exo_rew button in PlayerView

### ISSUE THREAD

**Issue by sajjadshahbazi** [google/ExoPlayer#9910]
I've overridden the controller layout of the player view. I've added another layout for controller view and overridden all of the widgets.

I can change the visibility of all of then the buttons, but I can't change the visibility of the exo_ffwd and exo_rew buttons.

    <com.google.android.exoplayer2.ui.PlayerView
        android:id="@+id/playerView"
        android:layout_width="0dp"
        android:layout_height="0dp"
        android:background="@color/black"
        app:controller_layout_id="@layout/exo_playback_control_view"
        app:fastforward_increment="10000"
        app:hide_during_ads="true"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent"
        app:resize_mode="fit"
        app:rewind_increment="10000"
        app:show_buffering="when_playing"
        />

layout/exo_playback_control_view (Segment of exo_rew and exo_ffwd )

    <ImageButton
        android:id="@+id/exo_rew"
        android:layout_width="@dimen/playerSizeItemController"
        android:layout_height="@dimen/playerSizeItemController"
        android:layout_marginEnd="@dimen/playerSpaceItemController"
        android:background="@drawable/bg_btn_tv_player_controller"
        android:scaleType="centerInside"
        app:layout_constraintBottom_toBottomOf="@+id/sPlay"
        app:layout_constraintEnd_toStartOf="@id/sPlay"
        app:layout_constraintTop_toTopOf="@+id/sPlay"
        app:srcCompat="@drawable/ic_tv_player_ten_sec_rev"
        tools:ignore="ContentDescription"
        android:nextFocusRight="@id/exo_ffwd"
        />

    <ImageButton
        android:id="@+id/exo_ffwd"
        android:layout_width="@dimen/playerSizeItemController"
        android:layout_height="@dimen/playerSizeItemController"
        android:layout_marginEnd="@dimen/playerSpaceItemController"
        android:background="@drawable/bg_btn_tv_player_controller"
        android:scaleType="centerInside"
        app:layout_constraintBottom_toBottomOf="@+id/sPlay"
        app:layout_constraintEnd_toStartOf="@+id/exo_rew"
        app:layout_constraintTop_toTopOf="@+id/sPlay"
        app:srcCompat="@drawable/ic_tv_player_ten_sec_forward"
        tools:ignore="ContentDescription"
        />

**Comment by sajjadshahbazi** [comment]
also, I define a style that visibility is gone , but does not work

**Comment by sajjadshahbazi** [comment]
I Set playerView.setShowFastForwardButton(false), but Doesn't work.

**Comment by tonihei** [comment]
I just tested a minimal example where I added `app:controller_layout_id="@layout/exo_playback_control_view"` to the `PlayerView`, referring to a custom layout file `layout/exo_playback_control_view` (which is just a plain copy of the original without modifications). In this minimal example, the buttons are shown by default and I can hide them by calling `playerView.setShowFastForwardButton(false)`.

I'd suggest you start with a minimal example again as close as possible to the original files and then add new customizations step by step to see which of your customizations break the functionality.
